sycl/test/esimd/block_load_store.cpp

```diff
@@ -63,17 +63,17 @@ kernel7(accessor<int, 1, access::mode::read_write, access::target::device> &buf)
     SYCL_ESIMD_FUNCTION {
   simd<int, 32> v1(0, 1);
   auto vals_view = v1.select<8, 1>(0);
-  block_store(buf, 0, vals_view);
+  block_store<int, 8>(buf, 0, vals_view);
```

Comment:
If keep the removed `simd_view vals` prototypes and have a call block_store(buf, 0, vals_view) without specifying `<T, N>` would the test pass or fail?